### PR TITLE
Allow making reference to global variables.

### PR DESCRIPTION
**Pull request by CodaFi**
Not sure if this should be partial.

### CODE REVIEW

**Comment by harlanhaskins** [comment]
Should this be in `Module` along with `function(named:)` and `type(named:)`?

**Comment by harlanhaskins** [comment]
Or should we move those both to `IRBuilder`?

File: Sources/LLVM/IRBuilder.swift [hunk 1]
```diff
@@ -989,6 +989,19 @@ public class IRBuilder {
   public func buildGlobalStringPtr(_ string: String, name: String = "") -> IRValue {
```
Comment (harlanhaskins) [inline Sources/LLVM/IRBuilder.swift:1026]: Let's go ahead and roll a change to make this return a `Global`

**Comment by CodaFi** [comment]
⛵️

**Pull request merged**